### PR TITLE
Filter/cufe

### PR DESCRIPTION
**Pull request by ZiaddAhmedd**
team name: cufe

### CODE REVIEW

File: results/yfcc-10M.png
Comment (harsha-simhadri) [inline results/yfcc-10M.png:1]: please remove the file from your PR. you are welcome to move it to the submission folder if you'd like

**Review by harsha-simhadri — CHANGES_REQUESTED** [review]

File: .github/workflows/neurips23.yml [hunk 1]
```diff
@@ -28,6 +28,8 @@ jobs:
             dataset: random-xs
             track: ood
           - algorithm: cufe
+            dataset: random-s
+            track: filter
             dataset: sparse-small
```
Comment (harsha-simhadri) [inline .github/workflows/neurips23.yml:33]: please add a separate entry for cufe filter and cufe sparse

**Comment by harsha-simhadri** [comment]
Are these results as expected 

```
cufe,"Faiss(('IVF4096,SQ8', {'nprobe': 4, 'mt_threshold': 0.0001}))",yfcc-10M,10,7005.467777167178,0.0,554.1706802845001,4980820.0,0.0,0.0,filter,0.721955
cufe,"Faiss(('IVF4096,SQ8', {'nprobe': 4, 'mt_threshold': 0.0003}))",yfcc-10M,10,6284.524075676667,0.0,554.1706802845001,4980820.0,0.0,0.0,filter,0.762003
cufe,"Faiss(('IVF4096,SQ8', {'nprobe': 10, 'mt_threshold': 0.0001}))",yfcc-10M,10,5312.795862267859,0.0,554.1706802845001,4980820.0,0.0,0.0,filter,0.826379
cufe,"Faiss(('IVF4096,SQ8', {'nprobe': 8, 'mt_threshold': 0.0003}))",yfcc-10M,10,5245.602730097057,0.0,554.1706802845001,4980820.0,0.0,0.0,filter,0.8361879999999999
cufe,"Faiss(('IVF4096,SQ8', {'nprobe': 12, 'mt_threshold': 0.0002}))",yfcc-10M,10,4755.515456714368,0.0,554.1706802845001,4980820.0,0.0,0.0,filter,0.8609640000000001
cufe,"Faiss(('IVF4096,SQ8', {'nprobe': 16, 'mt_threshold': 0.0001}))",yfcc-10M,10,4238.071924951995,0.0,554.1706802845001,4980820.0,0.0,0.0,filter,0.870327
cufe,"Faiss(('IVF4096,SQ8', {'nprobe': 16, 'mt_threshold': 0.0003}))",yfcc-10M,10,4033.9051602997056,0.0,554.1706802845001,4980820.0,0.0,0.0,filter,0.896015
cufe,"Faiss(('IVF4096,SQ8', {'nprobe': 16, 'mt_threshold': 0.00033}))",yfcc-10M,10,4013.6914950453065,0.0,554.1706802845001,4980820.0,0.0,0.0,filter,0.8986319999999999
cufe,"Faiss(('IVF4096,SQ8', {'nprobe': 30, 'mt_threshold': 0.00033}))",yfcc-10M,10,2917.1327145073938,0.0,554.1706802845001,4980820.0,0.0,0.0,filter,0.937862
cufe,"Faiss(('IVF4096,SQ8', {'nprobe': 32, 'mt_threshold': 0.00033}))",yfcc-10M,10,2806.7763921147266,0.0,554.1706802845001,4980820.0,0.0,0.0,filter,0.9411630000000001
```

**Comment by AbdelrahmanMohamed129** [comment]
no thats not the expected output can i make a modification?

**Comment by harsha-simhadri** [comment]
@AbdelrahmanMohamed129  Please do not change the parameters after the submission deadline. That is unfair to other participants.  Please limit any changes to small code fixes

**Comment by AbdelrahmanMohamed129** [comment]
I apologize I was unaware of that rule. What should I do now?

**Comment by maumueller** [comment]
@AbdelrahmanMohamed129 Please remove your latest commit from this branch.

**Comment by ZiaddAhmedd** [comment]
> @AbdelrahmanMohamed129 Please remove your latest commit from this branch.
> 
> Add more details about why the results posted by Harsha are "not expected".

@maumueller I reverted the commit of changing parameters and I just fixed the problem with neurips23.yml.
The results were too far from the expected I guess because machine specifications, but we were using the suggested machine from AWS.

**Comment by ZiaddAhmedd** [comment]
Is there something missing we need to do to get our submission merged?

File: .github/workflows/neurips23.yml [hunk 2]
```diff
@@ -28,8 +28,8 @@ jobs:
             dataset: random-xs
             track: ood
           - algorithm: cufe
-            dataset: sparse-small
-            track: sparse
+            dataset: random-s
+            track: filter
```
Comment (maumueller) [inline .github/workflows/neurips23.yml:32]: are you replacing the sparse CI with the filter CI here? Please fix.

**Review by maumueller — CHANGES_REQUESTED** [review]

File: .github/workflows/neurips23.yml (hunk 2 above)
Comment (AbdelrahmanMohamed129) [reply .github/workflows/neurips23.yml:32]: I just fixed it

**Comment by maumueller** [comment]
Alright, all CI runs are passing, I'm merging this.

**Review by maumueller — APPROVED** [review]

**Pull request merged**